src/qt/diagnosticsdialog.cpp

```diff
@@ -178,7 +178,7 @@ void DiagnosticsDialog::VerifyTCPPort() {
     connect(tcpSocket, SIGNAL(connected()), this, SLOT(TCPFinished()));
     connect(tcpSocket, SIGNAL(error(QAbstractSocket::SocketError)), this, SLOT(TCPFailed(QAbstractSocket::SocketError)));
 
-    tcpSocket->connectToHost("london.grcnode.co.uk", 32749);
```

Comment:
Change the port to use `GetListenPort()` from net.h and we'll cover both custom and testnet ports.